reddit/base.py

```diff
@@ -2,18 +2,14 @@
 from dotenv import load_dotenv, find_dotenv
 load_dotenv(find_dotenv())
 import os
+import config
```

Comment:
Please make sure that the `config` module is being imported here. Run the file and test it if you get any errors.